PayPal/src/main/java/com/braintreepayments/api/PayPalBrowserSwitchResult.java

```diff
@@ -0,0 +1,23 @@
+package com.braintreepayments.api;
+
+public class PayPalBrowserSwitchResult {
+
+    private BrowserSwitchResult browserSwitchResult;
+    private Exception error;
+
+    PayPalBrowserSwitchResult(BrowserSwitchResult browserSwitchResult) {
+        this.browserSwitchResult = browserSwitchResult;
+    }
+
+    PayPalBrowserSwitchResult(Exception error) {
+        this.error = error;
+    }
+
+    BrowserSwitchResult getBrowserSwitchResult() {
+        return browserSwitchResult;
+    }
+
+    Exception getError() {
+        return error;
+    }
```

Comment:
Do these need to be public w/ docstrings?

Reply:
No the idea with this result object is that the merchant doesn't have to do anything with it / be aware of any params and just passes it directly to the `PayPalClient#onBrowserSwitchResult` method, which will then handle any errors and call them back to the merchant. We could break it up and allow merchants to handle errors at each step, but this seems cleaner for integrations